From: Guillaume Delacour <gui@iroqwa.org>
Subject: Use the ncurses.h of libncursesw5-dev instead of libncurses5-dev
Forwarded: https://sourceforge.net/p/gptfdisk/code/merge-requests/1/

Index: gdisk/gptcurses.cc
===================================================================
--- gdisk.orig/gptcurses.cc
+++ gdisk/gptcurses.cc
@@ -22,7 +22,7 @@
 #include <iostream>
 #include <string>
 #include <sstream>
-#include <ncurses.h>
+#include <ncursesw/ncurses.h>
 #include "gptcurses.h"
 #include "support.h"
 
